aten/src/ATen/native/native_functions.yaml

```diff
@@ -2180,6 +2180,14 @@
   use_c10_dispatcher: full
   variants: method  # This is method-only to match the previous tensor API. In the future we could make this a function too.
 
+- func: movedim.intlist(Tensor self, int[] src, int[] dst) -> Tensor
```

Comment:
Since this function is a view, does it need an alias annotation? Maybe @albanD or @eellison?

Reply:
yea, `func: movedim.intlist(Tensor (a) self, int[] src, int[] dst) -> Tensor (a)`